testing_utilities/error_matchers_body.hpp

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+#include "testing_utilities/error_matchers.hpp"
+
+#include <ostream>
+
+#include "testing_utilities/numerics.hpp"
+
+namespace principia {
+namespace testing_utilities {
+namespace internal_error_matchers {
+
+using ::testing::MakeMatcher;
+using ::testing::MatcherInterface;
+using ::testing::MatchResultListener;
+
+template<typename Value>
```

Comment:
Put these classes in an unnamed namespace.

Reply:
No, this is a header; that's what the `internal` namespace is for.